identity-service/src/authMiddleware.js

```diff
@@ -53,22 +53,19 @@ async function authMiddleware (req, res, next) {
     if (!encodedDataMessage) throw new Error('[Error]: Encoded data missing')
     if (!signature) throw new Error('[Error]: Encoded data signature missing')
 
-    let walletAddress = recoverPersonalSignature({ data: encodedDataMessage, sig: signature })
+    const walletAddress = recoverPersonalSignature({ data: encodedDataMessage, sig: signature })
```

Comment:
this looks good. thx for cleanup. in future, probably good to have a separate PR even for something this small. this shouldn't break, but in the off chance it did, it would be spectacular and hard to find where it came from

Reply:
great point! thanks will keep in mind